### PR TITLE
Announce the SR2019 sign-up form.

### PR DESCRIPTION
**Pull request by Adimote**
This reverts commit 82bfd2ab81726d9447b94da81c9ce15b2d4fb5c4.

(because I'm an idiot and pushed straight to master)

### CODE REVIEW

File: SR2019/2019-02-09-competition-signup.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+preview-text: We're excited to open up volunteer sign-ups for Student Robotics 2019.
+---
+
+Hi,
+
+We're excited to open up volunteer sign-ups for Student Robotics 2019, which will take place on April 6th and 7th, at the University of Southampton Highfield campus.
+
+[Sign Up Now!][signup-form]
+
+We're expecting 37 teams, filling out this venue more than ever before, and we need some enthusiastic volunteers to help inspire the younger generation, and keep this event running as smooth as possible.
```
Comment (PeterJCLaw) [inline SR2019/2019-02-09-competition-signup.md]: is 37 really correct?

Comment (RealOrangeOne) [reply SR2019/2019-02-09-competition-signup.md]: Yes and no. In its current wording, yes. We hope to have 37 **teams** (if we can distribute the kits in time), however it's closer to 34 **schools**.

**Comment by Adimote** [comment]
Would be excellent to get an approval on this, ping: @PeterJCLaw and @RealOrangeOne

**Comment by Adimote** [comment]
> Just checking you also saw my comments about the form itself (in Slack)? I don't think they're blocking or required, just checking you'd seen them.

Now I have, and adjusted appropriately :)

**Comment by Adimote** [comment]
Scheduled to send in an hour!

https://mailchi.mp/d41b396685f5/sign-up-to-volunteer-atsr2019

**Pull request merged**